src/tests/ftest/pool/create_all_vm.py

```diff
@@ -20,9 +19,9 @@ def setUp(self):
         """Set up each test case."""
         super().setUp()
 
-        self.scm_avail_bytes = self.get_pool_available_bytes()
+        self.scm_avail_bytes = self.get_available_bits()
 
-    def get_pool_available_bytes(self):
+    def get_available_bits(self, ranks=None):
```

Comment:
nit: the bytes->bits rename seems a bit strange. The returned size is in bytes, no?

Reply:
Do not also understand why I changed bits in bytes.

- [ ] Refactoring of `get_available_bits()` into `get_available_bytes()`